content/en/docs/release-notes/release-notes-1.4.md

```diff
@@ -34,6 +32,247 @@ We have deprecated the following cert-manager APIs:
+the current Pod hardening best practices as defined by the Kyverno [pod-security
+restricted](https://kyverno.io/policies/pod-security/#restricted) policy.
+
+To pass the validation, the controller, webbook, and cainjector Pods are now
+running as non-root:
+
+```yaml
+apiVersion: v1
+kind: Pod
+spec:
+  securityContext:
+    runAsNonRoot: true
+```
+
+{{% pageinfo color="warning" %}}
+
+⛔️  If you are using custom containers that run as root with the Helm chart, you
+will need to set this back to `false`.
+
+{{% /pageinfo %}}
+
+## CA Issuer and `ca.crt`
+
+The CA issuer [now](https://github.com/jetstack/cert-manager/pull/3865) attempts
+to store the root CA instead of the issuing CA into the `ca.crt` field for
+issued certificates. All of the information which was previously available is
+still available: the intermediate should appear as part of the chain in
+`tls.crt`.
+
+{{% pageinfo color="warning" %}}
+
+⛔️  This is a change of behavior with the CA Issuer, make sure that the change in
+`ca.crt` does not impact you current use.
+
+{{% /pageinfo %}}
+
+# New Features
+
+## Support for the built-in CertificateSigningRequests
+
+It is [now](https://github.com/jetstack/cert-manager/pull/4064) possible to use
+the built-in Kubernetes [CertificateSigningRequest][] resources with
+cert-manager. The CA Issuer is currently the only supported issuer. The feature
+is experimental and can be enabled by adding a flag to the cert-manager
+controller. For example, with Helm:
+
+```sh
+helm install cert-manager jetstack/cert-manager \
+  --set extraArgs="{--feature-gates=ExperimentalCertificateSigningRequestControllers=true}"
+```
+
+Imagining that you have the following CA issuer:
+
+```yaml
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  namespace: default
+  name: my-ca-issuer
+spec:
+  ca:
+    secretName: ca-key-pair
+```
+
+You will make sure to be given the following role so that cert-manager can sign
+your CSR:
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+rules:
+- apiGroups:
+  - cert-manager.io
+  resources:
+  - signers
+  verbs:
+  - reference
+  namespaces:
+  - default
+  resourceNames:
+  - my-ca-issuer
+```
+
+You can create your Kubernetes CSR:
+
+```sh
+openssl genrsa -out test.key 2048
+kubectl apply -f - <<EOF
+apiVersion: certificates.k8s.io/v1
+kind: CertificateSigningRequest
+metadata:
+  name: test
+spec:
+  groups:
+  - system:authenticated
+  request: $(openssl req -new -key test.key -out /dev/stdout | base64 | tr -d "\n")
+  signerName: kubernetes.io/kube-apiserver-client
+  usages:
+  - client auth
+EOF
+```
+
+You will still need to manually approve the CSR since cert-manager does not
+approve CSRs by default
+([unlike](../../concepts/certificaterequest/#approval)
+CertificateRequests that get automatically approved by default):
+
+```sh
+kubectl certificate approve test
+```
+
+See [the documentation](../../usage/kube-csr) on how use the built-in
+CertificateSigningRequests with cert-manager.
```

Comment:
this link fails since it only exists in next-docs right now. When release-next is merged into master, the issue will go away. I don't know how to work around Prow regarding that though

Should I use the link 

```md
[the documentation](/next-docs/usage/kube-csr)
```

instead?

Reply:
> When release-next is merged into master

Since we are releasing today, do you think we can just somehow co-ordinate the merging so that we merge release-next into master before merging this PR?

Reply:
Also looking at the error message, might it just be the trailing slash?:

```
docs/release-notes/release-notes-1.4/index.html
  target is a directory, href lacks trailing slash --- docs/release-notes/release-notes-1.4/index.html --> ../../usage/kube-csr
```

Reply:
Lol once again I did not read the error message and assumed it was a broken link. Thank you for taking a look at the Prow build logs 🙏